File: desktop/src/app/livesituation/resetService.request.ts
/**
 * This class defines a model for requests to reset services in simulation mode in TraMS.
 */
export class ResetServiceRequest {

    private company: string;

    /**
     * Construct a new request to reset services.
     * @param company the company that we should reset services for.
     */
    constructor(company: string) {
        this.company = company;
    }
}

File: server/src/main/java/de/davelee/trams/server/request/ResetServiceRequest.java
package de.davelee.trams.server.request;

/**
 * This class is part of the TraMS Server REST API. It represents a request to reset services at the start of
 * a new day.
 * @author Dave Lee
 */
public class ResetServiceRequest {

    /**
     * Company which services should be reset for
     */
    private String company;

    public ResetServiceRequest() {
    }

    public ResetServiceRequest(String company) {
        this.company = company;
    }

    public String getCompany() {
        return company;
    }

    public void setCompany(String company) {
        this.company = company;
    }
}
